packages/react/src/index.tsx

```diff
@@ -37,39 +37,32 @@ export type VariantASProps<Config extends TConfig, VariantsObj> = {
       };
 };
 
+type MergeElementProps<T extends React.ElementType, P extends object = {}> = Omit<
```

Comment:
Could you please name the arguments in here to be more expressive (like element and props) as we're trying to make the types as readable as possible for easy debugging and to allow others to contribute to them easily.

this would also make it more in line with the rest of the types in the file ❤️

Reply:
thank you @hadihallak for the feedback )

Reply:
also, I used pascalCase to name them. is it that okay with you? I think it makes it easier to read